src/components/header/header.scss

```diff
@@ -0,0 +1,60 @@
+$primary-color: rgb(27, 41, 48);
+$logo-color: rgb(246, 205, 77);
+$border-color: rgb(150, 150, 150);
+$social-icon-size: 4rem;
+
+#header {
+  background: $primary-color;
+  display: flex;
+  height: 10rem;
+  color: #fff;
+}
+
+.logo {
+  margin: 1rem 0 1rem 0;
+}
+
+.logo path {
+  fill: $logo-color;
+}
+
+.logo svg {
+  padding: .75rem 0 .75rem 0;
+  height: 100%;
+}
+
+.social {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+}
+
+.social div {
+  margin-right: $social-icon-size/1.5;
+}
+
+.fa.fa-facebook {
+  color: #fff;
+  background: rgb(60, 90, 150);
+}
+
+.fa.fa-twitter {
+  color: #fff;
+  background: rgb(89, 173, 235);
+}
+
+.slack {
+  width: $social-icon-size;
+}
+
+.circle {
+  background-color: #333;
+  border-radius: $social-icon-size;
+  color: #fff;
+  display: inline-block;
```

Comment:
`inline-block` isn't really necessary here, it's getting inherited from `.fa` anyway.